tests/test_pipeline.py

```diff
@@ -219,6 +219,13 @@ def test_predict_triples(self):
             assert df.shape[0] == self.testing_mapped_triples.shape[0]
             assert {"head_id", "relation_id", "tail_id", "score"}.issubset(df.columns)
 
+    # TODO: needs matplotlib; cf. decorator from https://github.com/pykeen/pykeen/pull/948
```

Comment:
why don't you put that decorator right in this PR so we don't have to depend on the OGB thing, which still has a few questions